cli/client.go

```diff
@@ -736,9 +736,7 @@ func (c *AppClient) StartRobotPartShell(
 	}
 	robotClient, err := client.New(dialCtx, fqdn, logger, client.WithDialOptions(rpcOpts...))
 	if err != nil {
-		fmt.Fprintln(c.c.App.ErrWriter, err)
```

Comment:
This pattern was in a couple places across the CLI. Returning the error will cause it to be printed to `ErrWriter` and cause the CLI to exit with 1. We should just return here and elsewhere IMO.